platform/web/export/export_plugin.cpp

```diff
@@ -170,6 +170,7 @@ void EditorExportPlatformWeb::_fix_html(Vector<uint8_t> &p_html, const Ref<Edito
 	replaces["$GODOT_PROJECT_NAME"] = GLOBAL_GET("application/config/name");
 	replaces["$GODOT_HEAD_INCLUDE"] = head_include + custom_head_include;
 	replaces["$GODOT_CONFIG"] = str_config;
+	replaces["$GODOT_SPLASH"] = p_name + ".png";
```

Comment:
In a future PR, we could have Godot write a lossless WebP instead of PNG as it'll have a smaller file size.

Reply:
@Calinou Usually, games aren't shared directly by the player link, so it's not that bad.